src/lava/proc/plateau/process.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+
+import typing as ty
+from lava.magma.core.process.process import AbstractProcess
+from lava.magma.core.process.variable import Var
+from lava.magma.core.process.ports.ports import InPort, OutPort
+
+
+class Plateau(AbstractProcess):
+    """Plateau Neuron Process.
+
+    Couples two modified LIF dynamics. The neuron posesses two potentials,
```

Comment:
Thanks for providing such an extensive, well written docstring!

Is 'potential' the right word, or 'voltage'?